BubblesKotlin/app/src/main/java/com/example/android/bubbles/ReplyReceiver.kt

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright (C) 2020 The Android Open Source Project
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.example.android.bubbles
+
+import android.app.RemoteInput
+import android.content.BroadcastReceiver
+import android.content.Context
+import android.content.Intent
+import com.example.android.bubbles.data.ChatRepository
+import com.example.android.bubbles.data.DefaultChatRepository
+
+/**
+ * Handles the "Reply" action in the chat notification.
+ */
+class ReplyReceiver : BroadcastReceiver() {
+
+    companion object {
+        const val KEY_TEXT_REPLY = "reply"
```

Comment:
private?

Reply:
The key is passed to the notification in NotificationHelper so that the system fills the text in it.